src/Umbraco.Web.UI.Client/src/common/services/templatehelper.service.js

```diff
@@ -16,7 +16,7 @@
                 partialViewName = parentId + "/" + partialViewName;
             }
 
-            return "@Html.Partial(\"" + partialViewName + "\")";
+            return "@await Html.PartialAsync(\"" + partialViewName + "\")";
```

Comment:
It seems you are missing parenthesis to wrap this when using `await` keyword unlike when the static `Html` class is used.

Reply:
The parenthesis aren't required for it to work. I tested out and in the MS docs they use it without. I can add it though but I thought it just looked abit cleaner.

Reply:
If cleanliness is the goal, maybe interpolate the string, which removes the need to escape quotes? 

Related - a JS style guide would be awesome.

Reply:
Possibly but then it should probably be done for the rest of the file and that's a bit out of scope of this change.

Could always raise another issue for it.

Reply:
Yes, it seems you are right. I recall I tried in previous, but not sure why it didn't worked to me at that time.
Anyway I justed tested it in a project on v9.0.0 and it does seem to work :)